eAbsentee/form/templates/confirmationspanish.html

```diff
@@ -67,6 +68,10 @@ <h1> Su aplicación ha sido confirmada. </h1>
       <a href="https://www.facebook.com/voteabsenteeva">
         <i class="fa fa-facebook-square fa-2x"></i>
       </a>
+      &emsp;
+      <a href="instagram.com/voteabsenteeva">
```

Comment:
let's specify protocol etc to be consistent w/ FB icon
```suggestion
      <a href="https://www.instagram.com/voteabsenteeva">
```